secure_message/repository/modifier.py

```diff
@@ -76,11 +77,23 @@ def add_unread(message, user):
         return res
 
     @staticmethod
-    def del_unread(message, user):
+    def mark_message_as_read(message, user, session=db.session):
         """Remove unread label from status"""
         inbox = Labels.INBOX.value
         unread = Labels.UNREAD.value
         if inbox in message['labels'] and unread in message['labels'] and 'read_date' not in message:
-            Saver().save_msg_event(message['msg_id'], EventsApi.READ.value)
+            # Save to both events and secure_message table for now.  In future, the save to the
+            # events table will be removed.
+            try:
+                event = Events(msg_id=message['msg_id'], event=EventsApi.READ.value)
+                session.add(event)
+                result = SecureMessage.query.filter(SecureMessage.msg_id == message['msg_id']).one()
+                result.read_at = datetime.now(timezone.utc)
```

Comment:
Does this give anything different to `datetime.utcnow`?

Reply:
different if you look at the whole string, but it's the same when it gets saved to the database.  Changed it to that.